executorCommon/src/main/scala/dx/executor/JobMeta.scala

```diff
@@ -58,6 +60,9 @@ object JobMeta {
   val MaxConcurrentUploads = 8
   // max subjob input manifest document size for which a hash is used
   val MaxManifestJsLength = 100_000
+  // functions used to download manifest files
+  val ManifestDownloadDxda = "manifest_download_dxda"
```

Comment:
What is the difference between manifest and workflow manifest?

Reply:
Two different sets of files downloading to two different directories